APE_finance.rst

```diff
@@ -0,0 +1,143 @@
+
+Compensating an appropriate amount 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Process
++++++++
+In general (see next section), we put the funding level directly in the hands of the community and those making funding requests. We use a process where proposers suggest work and include a budget request which they think is required to achieve the suggested work. These suggestions are then put to the community for input and feedback. As part of the review process, the community is expected to take “value-for-money” into account. This mechanism places the trust to decide when compensation is unreasonably high or low to the community. In other cases, the Finance Committee will determine an appropriate amount taking into account the discussion and rationale below.
+
+Discussion and Rationale
+++++++++++++++++++++++++
+This is motivated by the fundamental tension that we, as a community, want to get the most out of the funds available, but we also want to avoid a “race to the bottom” where developers undercut others on price until the pay is too low to live on.
+Setting an appropriate level of pay for certain work is a difficult task, since a range of factors influences what people think is “fair”. This includes, to name just a few considerations,
+
+* external factors like the cost of living in a certain place or country, 
+* if people are paid as independent contractors (and thus have to factor vacation, sick time, medical insurance etc. into an hourly rate) or as a subaward to e.g. an academic institution (which usually set rates that include overheads and fringe benefits which are then provided  to the employee),
+* What monetary value one puts on experience (e.g. an undergraduate student vs. an experienced software developer).
+
+`NEP 48 <https://numpy.org/neps/nep-0048-spending-project-funds.html>`_ has extensive discussion and a good list of further references on this topic.
+
+We recognize though that the approach of leaving the approved budget entirely up to project members also has downsides: In particular, less experienced people may be tempted to suggest work packages with a compensation that is unfairly low for the work required to increase their perceived chances to obtain funding. We place trust in the community to recognize these short-comings of the process and to advise and mentor each other to propose realistic budgets that allow the proposed work to be completed with a fair rate of pay. As the diversity of proposers increases, we recognize that this approach will likely need to be modified and potentially be made more prescriptive. This may come in the form of a future APE or more incremental changes as experience grows.
```

Comment:
```suggestion
However, we recognize that the approach of leaving the approved budget entirely up to project members also has downsides. In particular, less experienced people may be tempted to suggest work packages with a compensation that is unfairly low for the work required to increase their perceived chances to obtain funding. We place trust in the community to recognize these short-comings of the process and to advise and mentor each other to propose realistic budgets that allow the proposed work to be completed with a fair rate of pay. As the diversity of proposers increases, we recognize that this approach will likely need to be modified and potentially be made more prescriptive. This may come in the form of a future APE or more incremental changes as experience grows.
```